### DISCUSSION TITLE
Significant differences in heat flux between FDS 6.7.1 and FDS 6.7.7

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12093] [General]
Hi all, 

I was performing some hydrocarbon pool fire FDS runs that involved measuring the heat flux on a distant object. I ran the same FDS file on FDS versions 6.7.1 and 6.7.7 that have generated significantly different results. The heat flux on the same object generates a ~7 kW/m^2 difference in heat fluxes (version 6.7.1 was a higher heat flux). Was there a significant difference between the solver for the two versions of FDS that would result in these differences? 

I'm going to run this same FDS file for version 6.8.0 to see how that one works out. If anyone could provide some insight, that would be greatly appreciated. Due to privacy issues I cannot post the entire FDS file, but I can post an abbreviated version of the file if needed.

**Comment by rmcdermo** [comment]
Do you have the same HRR with both 6.7.1 and 6.7.7?  And what is 7 kW/m2 integrated over the sphere compared to the total radiant emission?

**Reply by Tbuttts21** [reply]
Yes, the ratios of the average HRR are 1.003 so I don't think it's an issue with the fire itself.

**Comment by drjfloyd** [comment]
Ran the simple case below with 6.7.1 and 6.7.7.  Got fluxes of 4.6 and 3.8.   We fixed an error in the relative humidity calc which increased the default water vapor. The default pathlength of 0.1 m winds up with too much atmospheric absorption at large distances which magnifies the effect of a small change in H2O.

&HEAD CHID='radtest'/
&MESH XB=0,20,0,20,0,20,IJK=40,40,40/
&TIME T_END=0.001/
&SURF ID='HOT',COLOR='RED',TMP_FRONT=1000,TAU_T=0,EMISSIVITY=1/
&SURF ID='COLD',COLOR='BLUE',DEFAULT=T,EMISSIVITY=1/
&VENT PBX=20,SURF_ID='HOT'/
&DEVC QUANTITY='RADIATIVE HEAT FLUX',IOR=1,XYZ=0,10,10/

**Reply by drjfloyd** [reply]
If I zero out CO2 and H2O in the ambient (&MISC Y_CO2_INFTY=0,HUMIDITY=0/) I get the same result.

**Reply by Tbuttts21** [reply]
This case was run with the following settings: 

&HEAD CHID='Chid_Name'/
&TIME T_END=360.0/
&DUMP DT_PL3D=10.0, DT_RESTART=60.0, WRITE_XYZ=.TRUE., PLOT3D_QUANTITY(1:4)='HRRPUV','U-VELOCITY','V-VELOCITY','W-VELOCITY'/
&MISC HUMIDITY=47.0, TMPA=6.7/
&RADI NUMBER_RADIATION_ANGLES=600, PATH_LENGTH=11.0/
&WIND L=1.0E+6, Z_REF=10.0, SPEED=7.0, DIRECTION=0.0/

Do you think that same error would be observed with a PATH_LENGTH of 11?

**Reply by Tbuttts21** [reply]
Also, for clarification--was the error with version 6.7.1 or 6.7.7?

**Reply by drjfloyd** [reply]
Depending on target distance it may be a contributing factor. 

6.7.1 to 6.7.7 is many many changes to the source many of which could result in some change in the lean of the fire with the wind and the flame height. Changes in either of those would also have an impact on the flux.

**Reply by drjfloyd** [reply]
The error was in 6.7.1.

**Reply by Tbuttts21** [reply]
Thank you
